web/src/components/dashboard/QuickActionTile.tsx

```diff
@@ -0,0 +1,42 @@
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { MediaQueries } from "@/lib/PageSizes";
+import { useMediaQuery } from "@mui/material";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
+
+interface Props {
+  name: string;
+  url: string;
+  index: number;
+}
+
+export const QuickActionTile = (props: Props): ReactElement => {
+  const router = useRouter();
+  const isMobile = useMediaQuery(MediaQueries.isMobile);
+
+  const routeToAction = (): void => {
+    router.push(`${ROUTES.quickActions}/${props.url}`);
+  };
+
+  const getPadding = (): string => {
+    if (isMobile) {
+      return "0.25rem 0 0 0";
+    }
+
+    return props.index % 2 === 0 ? "0 0.25rem 0 0" : "0 0 0 0.25rem";
+  };
```

Comment:
Isn't this a bit of an anti pattern? Wouldn't it be better to do this as a matter of css

https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

<img width="850" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/16ec72bc-7924-410f-8a9f-bbdaf5ae293a">

Reply:
Is this not possible for some reason I'm not seeing at a glance?

Reply:
The "reason that you're not seeing" is that you are a Smartie and I am stuck in JS-brain. will do!